stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -66,6 +66,19 @@ public static void main(String[] args) {
     mainInternal(args, new FuseIOBench());
   }
 
+  @Override
+  public String getBenchDescription() {
+    return "This stress bench is designed for testing the reading throughput of Fuse-based "
+        + "POSIX API.\nTo write data, run `bin/alluxio runClass alluxio.stress.cli.fuse."
+        + "fuseIOBench --operation Write`\nTo read data, run `bin/alluxio runClass "
+        + "alluxio.stress.cli.fuse.fuseIOBench --operation Read`\nYou can further adjust "
+        + "the parameters specified above. Make sure the local path is successfully mounted\n"
+        + "For example, to read 100 files of size 100MB with 5 threads at local path /home/"
```

Comment:
Note that, files need to be written first before reading. For example, use `bin/alluxio runClass alluxio.stress.cli.fuse.fuseIOBench --local-path /mnt/alluxio-fuse --operation Write --num-files 100 --file-size 100m --threads 32 --bench-timeout 2h` to write the test files first and then use `bin/alluxio runClass alluxio.stress.cli.fuse.fuseIOBench --local-path /mnt/alluxio-fuse --operation Read --num-files 100 --file-size 100m --threads 32 --warmup 15s --duration 30s` to test the read throughput.